bookmarklet.js

```diff
@@ -1,32 +0,0 @@
-// This code is used to create the tooltip bookmarklet to show the mapping of
-// salesforce fields => fields displayed on the listing page
-// http://ted.mielczarek.org/code/mozilla/bookmarklet.html
-
-addTooltipsToChildren = function(children) { angular.forEach(children, function(e) {
```

Comment:
weird. no idea what this is. I guess some firefox extension?

Reply:
I even wonder what that was even supposed to do. It wasnt referenced anywhere, even on the initial commit that added it.

Reply:
strange! good to get rid of it then